scripts/util.py

```diff
@@ -3,30 +3,47 @@
 import re
 from logging import error
 from common import *
+from typing import Tuple
 
 
-def oss_fuzz_one_target(p, proj, fuzztime):
-    target, fuzzout = p
-    return subprocess.Popen(
-        [
-            "python3",
-            f"{OSSFUZZ}/infra/helper.py",
-            "run_fuzzer",
-            proj,
-            target,
-            f"\-max_total_time={fuzztime}",
-            "\-rss_limit_mb=0",
-            "--engine",
-            "libfuzzer",
-            "--corpus-dir",
-            fuzzout,
-        ],
-        stdout=subprocess.DEVNULL,
-        stderr=subprocess.DEVNULL,
-    )
+def oss_fuzz_one_target(
+    p: Tuple[str, str, str | None], proj: str, fuzztime: int
```

Comment:
the type of `dump` is `str | None` for optional dump,
should I change this type to `Maybe[str]` for better code style?
(something like rust `std::option::Option` is also in py now)

Reply:
Let's not worry about that too much.

Reply:
But I am more worried: are we using dyn lib or static lib to compile? Is all of them static; or hybrid? If hybrid, you should probably document which of them is dyn lib, which of them is static.

Reply:
I'll add a todo issue to document out lib usage. In case of this PR, some projects have their own dyn lib dependency (not our `reporter`), so they have to be run in docker.